doc/CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ if (DOXYGEN_FOUND)
 		"${CMAKE_CURRENT_BINARY_DIR}/Doxyfile"
 		@ONLY
 	)
+	file(GENERATE
```

Comment:
We need the command, since `file(GENERATE …)` [**does not** expand variables](https://cmake.org/Bug/view.php?id=16056):

> file(GENERATE) does not document that it expands variables. It doesn't expand them in either case. In fact it is not well-defined to do so because the generation does not occur until after configuration is complete when the variable definitions may no longer be available.

.